DISPLAY "(null)" invalid; disabling X11 forwarding
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcuda.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcurand.so locally
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
E tensorflow/stream_executor/cuda/cuda_driver.cc:491] failed call to cuInit: CUDA_ERROR_NO_DEVICE
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:153] retrieving CUDA diagnostic information for host: raiders2
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:160] hostname: raiders2
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:185] libcuda reported version is: Not found: was unable to find libcuda.so DSO loaded into this program
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:347] driver version file contents: """NVRM version: NVIDIA UNIX x86_64 Kernel Module  352.93  Tue Apr  5 18:18:24 PDT 2016
GCC version:  gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.1) 
"""
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:189] kernel reported version is: 352.93.0
I tensorflow/core/common_runtime/gpu/gpu_init.cc:81] No GPU devices available on machine.
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, localhost:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, raiders3:2210, raiders3:2211, raiders3:2212, raiders8:2213, raiders8:2214, raiders8:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2205
INFO:tensorflow:Learning rate: 0.001000, momentum: 0.300000
INFO:tensorflow:Sync mode!!!!!!
INFO:tensorflow:Compute groups : 1
INFO:tensorflow:SyncReplicas enabled: replicas_to_aggregate=16; total_num_replicas=16
INFO:tensorflow:Num nodes per compute groups: 16
INFO:tensorflow:Tokens per step: 16
INFO:tensorflow:2016-07-26 07:01:24.853170 Supervisor
E0726 07:01:40.757645849   68324 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2209': timeout occurred
E0726 07:01:40.757715396   68324 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2210': timeout occurred
E0726 07:01:40.757732215   68324 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2211': timeout occurred
E0726 07:01:40.757746147   68324 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2212': timeout occurred
E0726 07:01:41.662267186   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:01:41.662307473   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:01:41.662326166   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:01:41.662349263   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:01:44.926254060   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:01:44.926297131   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:01:44.926316806   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:01:44.926329612   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:01:47.926302407   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:01:50.926198060   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:01:50.926237745   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:01:50.926250947   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:01:53.926191209   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:01:57.150160555   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:01:57.150193687   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:02:00.150145134   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:02:03.150200019   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:02:06.150153125   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:02:06.150188557   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:02:15.150137841   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:02:18.150120131   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:02:21.798158591   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:02:21.798193155   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 07:02:37.322189087   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 07:02:40.322098993   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 07:02:47.650124061   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 07:02:50.650101092   68324 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
run-expr-32node.sh: line 23: 67885 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
